hack/chains/pipeline-quay-demo.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+source $(dirname $0)/_helpers.sh
+set -e
+
+title "Setting project"
+# Todo: Make it work in any project
+oc project tekton-chains
+
+QUAY_IMAGE=$1
+QUAY_SECRET_NAME=$2
+
+if [[ -z "$QUAY_IMAGE" ]] || [[ -z "$QUAY_SECRET_NAME" ]]; then
+  echo "Example usage:"
+  echo "  $0 quay.io/sbaird/chains-demo sbaird-chains-demo-pull-secret-name"
+  exit 1
+fi
+
+if ! kubectl get secret/$QUAY_SECRET_NAME -o name; then
+  echo "Can't find $QUAY_SECRET_NAME!"
+  echo "Please ensure it exists and try again!"
+  echo "For example:"
+  echo "  kubectl create -f sbaird-chains-demo-secret.yml"
+  exit 1
+fi
+
+title "Quay image"
+echo $QUAY_IMAGE
+
+title "Image push secret name"
+echo $QUAY_SECRET_NAME
+
+title "Suggested config for this demo:"
+$SCRIPTDIR/config.sh quay --dry-run
+
+title "Current config:"
+$SCRIPTDIR/config.sh
+
+title "To watch the chains controller logs:"
+echo "  kubectl logs -f -l app=tekton-chains-controller -n tekton-chains | sed G"
+
+pause
+
+title "Creating and configuring pipeline for demo"
+kubectl apply -f $SCRIPTDIR/pipeline-quay-demo.yaml
+
+title "Adding secret to tekton-chains service account"
+# Chains also needs the quay secret so it can push signatures
+# and (in future maybe) attestations
+kubectl patch sa pipeline -n tekton-chains \
```

Comment:
Not blocking, do we need to have the demo tied up to the tekton-chains project ?

Reply:
For now yes, but I have a todo somewhere to make it work in any project